src/display/annotation_helper.js

```diff
@@ -269,6 +269,45 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
     return container;
   }
 
+  function getHtmlElementForVideoAnnotation(item) {
```

Comment:
After #6770, all this code should be in a new class called `VideoAnnotationElement`. Please look at the current code to see how to structure this class. Creating an empty container, for instance, does not have to be done anymore as it is automatically available in `this.container` when you inherit from `AnnotationElement`.